tests/installation/module_registration/register_extensions_and_modules.pm

```diff
@@ -14,18 +14,18 @@ use warnings;
 use testapi qw(save_screenshot get_var);
 
 sub run {
-    my @scc_addons = split ',', get_var('SCC_ADDONS');
+    my @scc_addons = grep($_, split(/,/, get_var('SCC_ADDONS', '')));
     $testapi::distri->get_module_registration()->register_extension_and_modules([@scc_addons]);
     save_screenshot;
 
     # when some module (e.g. workstation extension) requires registration, provide separate code
     my $timeout = 60 * get_var('TIMEOUT_SCALE', 1);
     my $regcode = get_var('SCC_REGCODE_WE');
-    $testapi::distri->get_module_regcode()->add_separate_registration_code($regcode, $timeout);
+    $testapi::distri->get_module_regcode()->add_separate_registration_code($regcode, $timeout) if ($testapi::distri->get_module_regcode()->check_module_regcode_page);
```

Comment:
we don't have conditional logic in the test layer, please add all the product modules so we don't need this logic for now, in the future we can create different test module to handle this if needed or find better solution.

Reply:
At aarch64 and s390x, they don't have 'we' addon, so they don't have registration page.

Reply:
I suggest create new ticket for fix yast-mru-install-minimal-with-addons.yaml since code need change.
wdyt? @jknphy 
If you agree, @tinawang123 will revert change on yast-mru-install-minimal-with-addons.yaml

Reply:
for s390x and aarch64 please keep needle-based modules and file a progress ticket to tackle in the future, for x86_64 we can apply libyui with `register_extensions_and_modules`.

Reply:
I want to keep  s390x and aarch64 to use libyui with register_extensions_and_modules.  
I have done VR for them: 
aarch64: https://openqa.nue.suse.com/tests/9749601#
s390x: https://openqa.nue.suse.com/tests/9749599# (Failed at ticket poo#76813)
I have opened one ticket to fix register page problem: https://progress.opensuse.org/issues/118993
After this ticket fixed. Those cases can pass.